Yoast/wordpress-seo
# P2 423 extract method to helper

## Context


* We want to have one method that does the calculation.

## Summary


This PR can be summarized in the following changelog entry:

* Optimizes the code to have one method for retrieving the unindexed count.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Reset everything (indexables + prominent words)
* Verify that the notification for indexing SEO data is given.
* Verify that the indexing process still works
* Now remove between the 5 and 20 indexables and verify that these are calculated in the background.
* In the test helper set the plugin version to something below 14.5 and verify that the update routine works / won't give errors.

* Merge this pull request in a temporary premium branch 
* It should also work for the prominent words
   * A notification should be visible
   * A button to index in the link suggestions box should be given
   * A button to index on the tools page should be given.

## UI changes

## Documentation

## Quality assurance


Fixes #
